variables.tf

```diff
@@ -15,6 +15,9 @@ variable "DISK_SIZE_GB" {
   default = "500"
 }
 
+variable "IOPS" {
+  default = 3000
```

Comment:
This would be our first non-string terraform variable. Have you tested with different values and check those get reflected on AWS, right?

Reply:
Yes the values did get reflected